configure-places-in-the-sdk/places-extension/using-places-without-active-region-monitoring.md

```diff
@@ -0,0 +1,415 @@
+                    .setTransitionTypes(Geofence.GEOFENCE_TRANSITION_ENTER)
+                    .setCircularRegion(poi.getLatitude(), poi.getLongitude(), poi.getRadius())
+                    .setExpirationDuration(Geofence.NEVER_EXPIRE)
+                    .build();
+                Places.processGeofence(poiGeofence, Geofence.GEOFENCE_TRANSITION_ENTER);
+            }
+
+            // add the region to our new list of regions
+            updatedRegionsUserIsIn.add(poi.getIdentifier());
+        }
+    }
+
+    // update SharedPreferences with our new list of regions
+    SharedPreferences.Editor editor = getApplicationContext().getSharedPreferences("places", 0).edit();
+    editor.putStringSet("regionsUserIsAlreadyIn", updatedRegionsUserIsIn).apply();
+}
+```
+{% endtab %}
+
+{% tab title="iOS (Objective-C)" %}
+
+This code sample shows handling the result provided in the callback of `getNearbyPointsOfInterest:limit:callback:errorCallback:`, an `NSArray<ACPPlacesPoi *> *`:
+
+```objectivec
+- (void) handleUpdatedPOIs:(NSArray<ACPPlacesPoi *> *)nearbyPois {
+   // get the list of regions we know the user is already within from user defaults
+   NSArray *regionsUserIsCurrentlyWithin = [[NSUserDefaults standardUserDefaults]
+                                            arrayForKey:@"regionsUserIsAlreadyIn"];
+
+   // loop through new nearbyPoi and post entry events for pois that aren't already in our list
+   // also creating the new list of known regions that the user is in
+   NSMutableArray *updatedRegionsUserIsCurrentlyWithin = [@[] mutableCopy];
+   for (ACPPlacesPoi *poi in nearbyPois) {
+       // check if the user is in this poi
+       if (poi.userIsWithin) {
+           // the user is in the poi, now we need to make sure we haven't already recorded the entry event
+           if (![regionsUserIsCurrentlyWithin containsObject:poi.identifier]) {
+               CLCircularRegion *region = [[CLCircularRegion alloc] initWithCenter:CLLocationCoordinate2DMake(poi.latitude, poi.longitude)
+                                                                            radius:poi.radius
+                                                                        identifier:poi.identifier];
+               [ACPPlaces processRegionEvent:region forRegionEventType:ACPRegionEventTypeEntry];
+           }
+
+           // add the region to our updated list
+           [updatedRegionsUserIsCurrentlyWithin addObject:poi.identifier];
+       }
+   }
+
+   // update user defaults with the new list
+   [[NSUserDefaults standardUserDefaults] setObject:updatedRegionsUserIsCurrentlyWithin forKey:@"regionsUserIsAlreadyIn"];
+}
+```
+
+{% endtab %}
+
+{% tab title="iOS (Swift)" %}
+
+This code sample shows handling the result provided in the callback of `getNearbyPoints(_ ofInterest: CLLocation, limit: UInt, callback: (([ACPPlacesPoi]?) -> Void)?, errorCallback: ((ACPPlacesRequestError) -> Void)?)`, an `[ACPPlacesPoi]`:
```

Comment:
Please reword to:
""The following code sample shows the handling of the result that was provided..."